app-admin/lnav/lnav-0.10.1.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+DESCRIPTION="A curses-based tool for viewing and analyzing log files"
+HOMEPAGE="https://lnav.org"
+SRC_URI="https://github.com/tstack/${PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="unicode test"
+RESTRICT="!test? ( test )"
+
+DEPEND="
+	test? ( dev-cpp/doctest )
+"
+
+RDEPEND="
+	app-arch/bzip2:0=
+	app-arch/libarchive:=
+	>=dev-db/sqlite-3.9.0
+	dev-libs/libpcre[cxx]
+	>=net-misc/curl-7.23.0
+	sys-libs/ncurses:=[unicode(+)?]
+	sys-libs/readline:0=
+	sys-libs/zlib:0="
+DEPEND="$DEPEND $RDEPEND"
```

Comment:
Why not just define RDEPEND then DEPEND? Also, ${FOO} [style](https://projects.gentoo.org/qa/policy-guide/ebuild-format.html#pg0101). Will change for you. Thank you!